mods/cnc/weapons/missiles.yaml

```diff
@@ -237,6 +240,7 @@ TowerMissile:
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
 	Warhead@3Eff: CreateEffect
 		Explosions: med_frag
+		ImpactSounds: xplobig4.aud
```

Comment:
I don't think this works for the AGT as it is currently balanced. The explosion from the first missile hasn't even finished before the next one starts playing!

Reply:
You are right. Its fire rate is too high. Should I remove `xplobig4` and rebase again?

Reply:
That would probably best, but it would be good to get some opinions from some of our active TD players IMO.